### PR TITLE
Add 1.10 tests

### PR DESCRIPTION
**Pull request by hellozee**
~There is a single failure in `security/custom_ca_root` on the `TestMultiRootSetup` which I am yet to figure out, other than that rest seems fine.~

### CODE REVIEW

File: tetrateci/version_check.py [hunk 1]
```diff
@@ -3,9 +3,10 @@
 import sys, os
 
 version_matrix = {
-    "1.7" : {"1.16", "1.17", "1.18"},
-    "1.8" : {"1.16", "1.17", "1.18", "1.19"},
-    "1.9" : {"1.17", "1.18", "1.19", "1.20"},
+    "1.7": {"1.16", "1.17", "1.18"},
+    "1.8": {"1.16", "1.17", "1.18", "1.19"},
+    "1.9": {"1.17", "1.18", "1.19", "1.20"},
```
Comment (deva26) [inline tetrateci/version_check.py:8]: Does it not work on k8s 1.16?

File: tetrateci/version_check.py [hunk 2]
```diff
@@ -3,9 +3,10 @@
...
+    "1.10": {"1.18", "1.19", "1.20", "1.21"},
```
Comment (deva26) [inline tetrateci/version_check.py:9]: Does it not work on k8s 1.16?

**Review by deva26 — APPROVED** [review]

File: tetrateci/version_check.py (hunk 1 above)
Comment (hellozee) [reply tetrateci/version_check.py:8]: Not supported according to [this](https://istio.io/v1.9/docs/setup/platform-setup/)

File: tetrateci/version_check.py (hunk 2 above)
Comment (hellozee) [reply tetrateci/version_check.py:9]: Same [here](https://istio.io/latest/docs/setup/platform-setup/)

**Pull request merged**